components/dropdown/dropdown-content-styles.js

```diff
@@ -30,17 +35,40 @@ export const dropdownContentStyles = css`
 		opacity: 0.9;
 	}
 
-	:host([opened]) {
-		-webkit-animation: var(--d2l-dropdown-animation-name) 300ms ease;
-		animation: var(--d2l-dropdown-animation-name) 300ms ease;
-		display: inline-block;
+	@media (min-width: 616px) {
+		:host([opened]) {
+			-webkit-animation: var(--d2l-dropdown-animation-name) 300ms ease;
```

Comment:
I _think_ the `-webkit-animation` property was only needed for Safari 8 and earlier.  I think some of the styles in here are quite old, and are probably being copied in a multitude of places.  Same goes for `-webkit-keyframes` and `-webkit-transform`. It might be nice to avoid them.